documentation/todo-app/src/components/todoMain.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import TodoItems from './todoItems';
+import './todo_main.css'
+
+export default class TodoMain extends Component {
+    render() {
+        const todos = [
```

Comment:
You should declare this variable outside your render function. As you have it right now, the variable is redeclared each time the render function runs. This is a small issue, but would have performance issues in some cases.